docker-deploy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
```

Comment:
It's just a better demarkation of where things happen. Build in the script and deploy in the deployment step.

That said in the context of the way you are running your tests it doesn't make a lot of difference, so I moved it into the deploy script.

I can't figure out why the tests are hanging though. As far as I can tell the only difference that could affect that is that it's now running on their legacy infrastructure (needed to build docker images for now) the tests also won't run for me locally.